lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/MiniMapController.java

```diff
@@ -0,0 +1,94 @@
+package nl.tudelft.lifetiles.graph.controller;
+
+import java.util.List;
+
+import javafx.geometry.Insets;
+import javafx.scene.control.ScrollBar;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.paint.CycleMethod;
+import javafx.scene.paint.LinearGradient;
+import javafx.scene.paint.Stop;
+import nl.tudelft.lifetiles.graph.model.GraphContainer;
+import nl.tudelft.lifetiles.graph.view.MiniMapScrollPaneSkin;
+
+/**
+ * The controller of the mini map.
+ *
+ * @author Joren Hammudoglu
+ *
+ */
+public final class MiniMapController {
+
+    /**
+     * The scroll bar to display the minimap on.
+     */
+    private final ScrollBar scrollBar;
+
+    /**
+     * The graph model.
+     */
+    private final GraphContainer model;
+
+    /**
+     * Create a new mini map controller.
+     *
+     * @param scrollPane
+     *            the scroll pane to display the minimap in.
+     * @param model
+     *            the graph model
+     */
+    public MiniMapController(final ScrollPane scrollPane,
+            final GraphContainer model) {
+        this.model = model;
+
+        MiniMapScrollPaneSkin skin = new MiniMapScrollPaneSkin(scrollPane);
+        scrollPane.setSkin(skin);
+        scrollBar = skin.getHorizontalScrollBar();
+    }
+
+    /**
+     * Draw the miniMap onto the horizontal scrollbar.
+     */
+    public void drawMiniMap() {
+        Background background = getMiniMapBackground();
+        scrollBar.setBackground(background);
+    }
+
+    /**
+     * Get the minimap scrollbar.
+     *
+     * @return the scrollbar
+     */
+    public ScrollBar getMiniMap() {
+        assert scrollBar != null;
+        return scrollBar;
+    }
+
+    /**
+     * Gets the MiniMap background gradient.
+     *
+     * @return the minimap background
+     */
+    private Background getMiniMapBackground() {
+        double buttonWidth = javafx.scene.text.Font.getDefault().getSize();
```

Comment:
What is the reason that you don't use import but rather directly reference to the static value?